src-docs/src/views/emotion/canopy.tsx

```diff
@@ -106,9 +102,9 @@ const View3 = () => {
 
 const View2 = () => {
   const overrides = {
-    [COLOR_MODE_KEY]: {
+    colors: {
       light: {
-        success: computed(['colors.primary'], () => '#85E89d'),
+        success: computed(() => '#85E89d', ['colors.primary']),
```

Comment:
Just double-checking that this doesn't actually need to be `computed` because it seems to just be returning a straight hex value?
```suggestion
        success: '#85E89d',
```

Reply:
Nope. Just left over from some check I was doing.